web/locales/en.json

```diff
@@ -42,6 +42,9 @@
         "noDataAtTimestamp": "Data is temporarily unavailable for the selected time",
         "noLiveData": "Live data is temporarily unavailable for this area",
         "dataIsDelayed": "Data for this region can be up to %s hours delayed. Live data cannot be displayed.",
+        "dataIsEstimated": "Data for this region is delayed. Real-time values are estimated. Read more about our estimation methodology <a href=\"https://github.com/electricityMap/electricitymap-contrib/wiki/Estimation-methods\" target=\"_blank\">here</a>.",
+        "estimatedDisclaimer": "The real-time data is estimated based on the data from the source and the assumptions of the model. The data is not guaranteed to be accurate.",
+        "helpUs": "Help us identify the problem by taking a look at the <a href=\"%s\" target=\"_blank\">runtime logs</a> or contact the data provider.",
```

Comment:
Source -> data provider

Reply:
@Kongkille any strong feeling here?

Reply:
I think this is fine for now :) 

@livmontmerle any thoughts on the copy for the feature?